docs/community.md

```diff
@@ -189,6 +189,34 @@ tags:
 ---
 ```
 
+### Check Content Formatting
+
+There have been some differences in how certain IDE’s such as Visual Studio code and mkdocs render markdown.  What this means is that when creating a page in Forensicswiki, some content looks fine in VS code but then once committed to Github the content is not displaying properly.  To ensure content is formatted correctly, you could use a local mkdocs install which has a built in development server.
```

Comment:
I personally would remove the double spaces, Markdown will not render them anyway (AFAIK).